Makefile

```diff
@@ -121,6 +121,7 @@ distclean: clean
 	docker volume rm -f $(GO_CACHE_VOLUME_NAME)
 	$(call rmi-if-exists,$(FIRECRACKER_CONTAINERD_TEST_IMAGE):$(DOCKER_IMAGE_TAG))
 	$(call rmi-if-exists,localhost/$(PROTO_BUILDER_NAME):$(DOCKER_IMAGE_TAG))
+	$(FIRECRACKER_DIR)/tools/devtool distclean
```

Comment:
Curious, if we should just call `firecracker-clean` target when someone does `make distclean`?

Reply:
`devtool distclean` and `make firecracker-clean`? Sounds good to me.